packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -215,7 +215,7 @@ export class DatabaseCluster extends DatabaseClusterRef {
         }
 
         this.defaultPortRange = new ec2.TcpPortFromAttribute(this.clusterEndpoint.port);
```

Comment:
Does `defaultPortRange` really need to be a property now?

Reply:
We still do static typing one-way. We still force the destination to have a default port by forcing it to be of type `IDefaultConnectable` (which exposes this property).